test/t8_test_forest_commit.cxx

```diff
@@ -127,7 +127,12 @@ t8_test_forest_commit (int cmesh_id)
   min_level = t8_forest_min_nonempty_level (cmesh, scheme);
   /* Use one level with empty processes */
   min_level = SC_MAX (min_level - 1, 0);
-  for (level = min_level; level < min_level + 3; level++) {
+#ifdef T8_ENABLE_LESS_TESTS
+  const int levelup = 2;
+#else
+  const int levelup = 3;
+#endif
+  for (level = min_level; level < min_level + levelup; level++) {
```

Comment:
Was there a particular reason to change the levelup to 2?

I would rather leave it at 3. Since this level is not like the other as in "the original uniform level of the test mesh".
It specifies a refinement level for testing the adapt and balance routine.
Especially for the balance routine it is important to refine a few levels in order to catch possible boundary cases of balance (like neighboring elements of large level diff).

Hence, i would not touch this parameter for the less tests.

Reply:
The reason was a time saving. I noticed that the 3rd levelup/forloop takes a lot of time. Especially the forest_commit takes much less time this way. To get a similar benefit, the cmesh test cases have to be reduced significantly.

t8_test_forest_commit takes about 80% of the time of the t8code test according to my rough measurements.

I understand your concerns however. I have not thought about that. 
The changes are reversed and once #17 is fixed, I will adjust the macros accordingly.

Reply:
Thanks for clearing this up.
Its great that you did the timings of the tests and i see your reasoning.

I think your suggestion to wait for #171 and checking for the reduction in runtime is good.
Should this test case then still take too long we can reevaluate